### DISCUSSION TITLE
A respring button somewhere in Zebra?

### DISCUSSION THREAD

**Discussion by ghost** [wstyres/Zebra#1783] [Ideas]
Sometimes when my phone battery gets low or I need to test one of my apps for bugs, I turn off tweak injection and respring. The problem is I always need to either upgrade or reinstall something to respring back with tweaks (or use that one terminal command I can't ever remember), so I think it would be helpful to have a button to just respring quickly somewhere in the app.

**Comment by absidue** [comment]
Hi BenSova

You thanks for the idea, it's already possible by going to Settings > Reset > Restart SpringBoard inside the Zebra app.
